tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -140,189 +125,47 @@ echo "whl file path = ${WHL_PATH}"
 # Install, in user's local home folder
 echo "Installing pip whl file: ${WHL_PATH}"
 
-# Call pip install on the whl file. We are doing it without the --upgrade
-# option. So dependency updates will need to be performed separately in
-# the environment.
-${PYTHON_BIN_PATH} -m pip install -v --user ${WHL_PATH} \
+# Create temporary directory for install test
+VENV_DIR="${PIP_TEST_ROOT}/venv"
+rm -rf "${VENV_DIR}" && mkdir -p "${VENV_DIR}"
```

Comment:
Does pip inside virtualenv work reliably without deleting the whole env? It would be nice for caching. We would reinstall only tf and dependencies only if their version would change. But I'm not sure if it would be reliable.